mmaction/datasets/pipelines/formating.py

```diff
@@ -371,3 +372,31 @@ def __repr__(self):
         repr_str = self.__class__.__name__
         repr_str += f"(input_format='{self.input_format}')"
         return repr_str
+
+
+@PIPELINES.register_module()
+class FormatNtuPose:
+
+    def __init__(self, input_format):
+        self.input_format = input_format
```

Comment:
The input_format is not used.

Reply:
The value of `input_format` should also be checked like `assert input_format in [xxx, xxx]`